src/containers/profile/profile.component.jsx

```diff
@@ -20,7 +20,7 @@ const Profile = (props) => {
       addUsernameToStateThunk(userWithUsername);
       uploadProfileAvatarThunk({ image: imageAsFile, username });
 
-      //TODO: Need to show user that this is successfull or has failed
+      //TODO: Need to show user that this is successful or has failed
```

Comment:
Whats the suggestion here? I feel as though the redirect does this?

Reply:
What do you mean exactly? Like if we take them to the home page it's good? But we take them to the homepage regardless of whether the API call is made successfully